src/components/Header.jsx

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  AppBar,
+  Divider,
+  Icon,
+  IconButton,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  SwipeableDrawer,
+  Toolbar,
+  Typography
+} from "@material-ui/core";
+import MenuIcon from "@material-ui/icons/Menu";
+import { withStyles } from "@material-ui/core/styles";
+import { Link } from "react-router-dom";
+
+const styles = {
+  appBar: {
+    backgroundColor: "#EEE",
+    color: "#000000"
+  },
+  udescColor: {
+    color: "#0a9854"
+  },
+  flex: {
+    flexGrow: 1
+  },
+  menuButton: {
+    marginLeft: -10,
+    marginRight: 15
+  },
+  menuList: {
+    width: 250
+  },
+  menuItem: {
+    textDecoration: "none"
+  }
+};
+
+class Header extends React.Component {
+  state = {
+    menu: false
+  };
+
+  toggleMenu = open => () => {
+    this.setState({
+      menu: open
+    });
+  };
+
+  render() {
+    const { classes } = this.props;
+
+    const menuItems = (
+      <div className={classes.menuList}>
+        <Typography
+          variant="display1"
+          align="center"
+          className={classes.udescColor}
+        >
+          Bem vindo, fudido!
+        </Typography>
+        <Divider />
+        <Link to="/" className={classes.menuItem}>
+          <ListItem button>
+            <ListItemIcon>
+              <Icon style={styles.udescColor}>home</Icon>
+            </ListItemIcon>
+            <ListItemText primary="Início" />
+          </ListItem>
+        </Link>
+        <Divider />
+        <Link to="/talks" className={classes.menuItem}>
```

Comment:
We can create `components` one for each link / button, can be done in this same file